doc/source/whatsnew/v0.25.0.rst

```diff
@@ -434,6 +434,8 @@ Other API Changes
 - The ``arg`` argument in :meth:`pandas.core.groupby.DataFrameGroupBy.agg` has been renamed to ``func`` (:issue:`26089`)
 - The ``arg`` argument in :meth:`pandas.core.window._Window.aggregate` has been renamed to ``func`` (:issue:`26372`)
 - Most Pandas classes had a ``__bytes__`` method, which was used for getting a python2-style bytestring representation of the object. This method has been removed as a part of dropping Python2 (:issue:`26447`)
+- The `.str`-accessor has been disabled for 1-level :class:`MultiIndex` (almost all methods were previously broken for this case),
```

Comment:
the parenthetical is not necessary

did this actually work at some point?

Reply:
The methods could be called but produced garbage, see #23679.